mkdocs.yml

```diff
@@ -8,6 +8,7 @@ nav:
     - 'Default Plugins': default_plugins.md
     - 'Writing Your Plugin': writing_your_plugin.md
     - 'Building ILLIXR': building_ILLIXR.md
+    - 'ILLIXR Debugging Tips': debugging_ILLIXR.md
```

Comment:
Is this supposed to point to an empty file?

Reply:
That was a mistake.

Reply:
debugging_illixr.md?
Also, please change building_ILLIXR to building_illixr -- I forgot to do it in another PR.